packages/phone-number-privacy/signer/azure-templates/container-parameters.json

```diff
@@ -0,0 +1,92 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "containerName": {
+            "value": "TODO"
+        },
+        "location": {
+            "value": "TODO"
+        },
+        "imageType": {
+            "value": "Public"
+        },
+        "imageName": {
+            "value": "us.gcr.io/celo-testnet/celo-monorepo:phone-number-privacy-rc1"
+        },
+        "osType": {
+            "value": "Linux"
+        },
+        "numberCpuCores": {
+            "value": "2"
+        },
+        "memory": {
```

Comment:
Nit: Can you add units to this? "memoryInGB"

Reply:
I didn't hand write this, it's what was exported by the Azure portal

Reply:
I'll take my concerns up with the Azure team 🙃